d912pxy/d912pxy_extras.cpp

```diff
@@ -87,13 +89,17 @@ void d912pxy_extras::Init()
 	{
 		d912pxy_config_value_dsc* iter = d912pxy_s.config.GetEntryRaw(d912pxy_config_value(configIndex));
 
-		if ((wcsstr(iter->name, L"enable_") || wcsstr(iter->name, L"show_")) && (wcscmp(iter->value,L"0") != 0))
+		if ( wcsstr(iter->name, L"show_") && (wcscmp(iter->value,L"0") != 0))
 		{
-			overlayShowMode = eoverlay_show;
+			bShowMainWindow = true;
 			break;
 		}
 	}
 
+	//Sets Config Editor window to visible if enabled when there's no need for the main window
+	if ((bShowMainWindow == false) && (bEnableConfigEditor == true))
+		bShowConfigEditor = true;
```

Comment:
then if you close editor window, there will be no way to open it back up, so it should be 
bShowMainWindow |= bEnableConfigEditor;

Reply:
Wouldn't that cause it to always start open even when the main window is present?
If the concern is the user closing the window when there's no main window the best approach I can think is a trenary operator on the window parameter.